src/components/autocomplete/js/autocompleteController.js

```diff
@@ -466,9 +466,9 @@ function MdAutocompleteCtrl ($scope, $element, $mdUtil, $mdConstant, $mdTheming,
         select(ctrl.index);
         break;
       case $mdConstant.KEY_CODE.ESCAPE:
+        event.preventDefault(); // Prevent browser from always clearing input
```

Comment:
I think I'm confused now 😕 

@clshortfuse So, you are saying that before this fix, there were some cases where the esc key was clearing the input when we didn't want it to. Can you give me an example?

@DevVersion What was I right about? I was stating that my initial thinking was wrong/incorrect 😄